components/profile/PersonalLinksSection.js

```diff
@@ -28,6 +28,15 @@ const PersonalLinkInput = ({ type, links, setLinks }) => {
         setLinks({ type, data: { value, valid: isValid } })
         break
       }
+      case 'aclanthology': {
+        const isValid =
+          isValidURL(value) && value.startsWith('https://aclanthology.org/people/')
```

Comment:
The API has a slightly different rule for validation, the link must also have some text that comes after "people/". Maybe we could just add an additional check `&& value.length > 32`

Reply:
changed to use the same regex as in api so that the general url regex check can be skipped